src/blackboxTests/src/test/kotlin/org/vaccineimpact/api/blackboxTests/tests/Coverage/PopulateCoverageTests.kt

```diff
@@ -21,32 +21,15 @@ class PopulateCoverageTests : CoverageTests()
 {
     private val requiredPermissions = PermissionSet("*/coverage.write")
 
-    @Test
-    fun `can populate coverage`()
-    {
-        setup()
-        validate("/touchstones/$touchstoneVersionId/coverage/", method = HttpMethod.post) withRequestSchema {
-            CSVSchema("CoverageIngestion")
-        } sending {
-            csvData
```

Comment:
This was writing CSV data directly to the request body. Now that we require the "description" field also, we will only accept multipart form data.